cmd/runner/studio_run.go

```diff
@@ -893,6 +1023,8 @@ type studioRunOptions struct {
 //
 func studioRun(ctx context.Context, opts studioRunOptions) (err kv.Error) {
 
+	fmt.Printf(">>>>>>> START studioRun %+v\n", opts)
```

Comment:
I made the assumption these types of prints are there until your happy with it all working and then will go, or be turned into maybe logger.Trace ?

Reply:
Debug prints cleaned out.